shell.nix

```diff
@@ -5,6 +5,7 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [
     binutils
+    cachix
```

Comment:
should this be a build input for `stack --nix`?

Reply:
See DisciplinaOU/disciplina-nixops@627e9ba7981b94e9b8bc63d8e96bda93be1fbc01.